networkmonitor/src/main/java/ca/rmen/android/networkmonitor/app/dbops/backend/export/kml/KMLWriter.java

```diff
@@ -73,7 +73,7 @@ public void writeHeader() {
 
     /**
      * Write a single Placemark
-     * 
+     *
```

Comment:
Since KMLWriter only has whitespace changes (after the reverting), could you just revert the changes completely?  One less file in the PR :)